src/test/java/ioanandroflorea/tests/Tema8/pageObjects/DisabledDynamicButtonsPage.java

```diff
@@ -0,0 +1,67 @@
+package ioanandroflorea.tests.Tema8.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class DisabledDynamicButtonsPage {
+
+    ChromeDriver driver;
+    WebDriverWait wait;
+
+    public DisabledDynamicButtonsPage(ChromeDriver driver) {
+        this.driver = driver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
```

Comment:
Try to increase the max time for wait. Try with a higher value than 5 and see which one make the test pass. Thanks!